### PR TITLE
Refactoring tgmath.swift tests

### PR DESCRIPTION
**Pull request by frootloops**
#### What's in this pull request?

Refactoring tgmath.swift tests

Please check it on Linux

### CODE REVIEW

File: test/1_stdlib/tgmath.swift [hunk 1]
```diff
@@ -108,390 +109,211 @@ var g1, g2: CGFloat
...
-// CHECK-NEXT: tan 0.100334672085451 0.100335 tan
-
-
-d1 = acosh(dx)
-f1 = acosh(fx)
-g1 = acosh(gx)
-print3("acosh", d1.isNaN, f1.isNaN, g1.isNaN)
-// CHECK-NEXT: acosh true true acosh
-// Note: We cannot check the output with "nan nan" using String interpolations
-// because glibc on Linux outputs the result as "-nan" instead of "nan".
```
Comment (rintaro) [inline test/1_stdlib/tgmath.swift]: Could you apply this on `acosh` test?

**Comment by gribozavr** [comment]
@swift-ci Please test

**Comment by frootloops** [comment]
@rintaro done

**Comment by gribozavr** [comment]
@swift-ci Please test

**Comment by frootloops** [comment]
@gribozavr It has failed on the linux platform. Could you give me some advice how can I fix it?

**Comment by gribozavr** [comment]
I believe this is due to an existing issue in swiftpm.  We will retry the CI when swiftpm is in better shape.

**Comment by gribozavr** [comment]
@swift-ci Please test Linux

**Comment by frootloops** [comment]
@gribozavr so what we gonna do?

**Comment by gribozavr** [comment]
@frootloops That problem is solved now, and the test was run and it failed:

```
stdout>>> check failed at /home/buildnode/jenkins/workspace/swift-PR-Linux/swift/test/1_stdlib/tgmath.swift, line 133
stdout>>> expected: "acosh nan nan acosh" (of type Swift.String)
stdout>>> actual: "acosh -nan -nan acosh" (of type Swift.String)
[     FAIL ] TGMath.Unary functions
```

**Comment by frootloops** [comment]
@gribozavr I see. It was an outdated test case. I have remove it and we can try again.

**Comment by tkremenek** [comment]
@swift-ci please test

**Comment by gribozavr** [comment]
@swift-ci Please test

**Comment by gottesmm** [comment]
(I think the tree might have been broken?)

@swift-ci Please test OS X Platform

**Comment by tkremenek** [comment]
@swift-ci please test

**Pull request merged**